src/screens/Subscriptions/AfterUnsubscribe.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { createRef } from 'react';
+import { Wrapper, Header, BodyCopy } from './Subscriptions.style';
+import { EventAction, EventCategory } from 'components/Analytics';
+import ExternalLink from 'components/ExternalLink';
+import { LargeFilledButton } from 'components/ButtonSystem';
+
+const AfterUnsubscribe = () => {
+  const params = new URLSearchParams(window.location.search);
+  const email = params.get('email') || '';
+  const formRef = createRef<HTMLFormElement>();
+
+  const formAction = getFormActionLink(email);
```

Comment:
it's prbly not necessary to do `getFormActionLink(email);` (which in turn does the `createsend.com` call) until the person actually clicks the subscribeClick link. so down below you can do:
```
if (formRef?.current) {
  const formAction = await getFormActionLink(email);
  formRef.current.action = await formAction;
  ...
```

Reply:
Ok, no worries - sgtm!